general-standards/README.md

```diff
@@ -1,24 +1,76 @@
 # AlliedCrowds General Standards
+
 > Let's play Russian Roulette:
 > `[ $[ $RANDOM % 6 ] == 0 ] && rm -rf / || echo *Click*`
 
+## Feature Lifecycle
+
+These are the stages that a new feature undergoes before being delivered. Collectively these can be considered "feature implementation":
+
+- **Initial conversation/Expression of need**: This is where the feature/issue is first expressed, it can be on a call, as a message, over email or in an issue itself
+- **Issue creation**: This is where it gets logged in the PM software
```

Comment:
I think this is a good place to make predictions about how long the set of issues will take. I was going to start logging this in the form of notes in the project todo column.